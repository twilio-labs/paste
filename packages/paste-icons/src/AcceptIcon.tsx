/**
 * This file was automatically generated with @twilio-labs/svg-to-react
 */
import * as React from 'react';
import {useUID} from '@twilio-paste/uid-library';

import {IconWrapper} from './helpers/IconWrapper';
import type {IconWrapperProps} from './helpers/IconWrapper';

export interface AcceptIconProps extends IconWrapperProps {
  title?: string;
  decorative: boolean;
}

const AcceptIcon = React.forwardRef<HTMLElement, AcceptIconProps>(
  ({as, display, element = 'ICON', size, color, title, decorative}, ref) => {
    const titleId = `AcceptIcon-${useUID()}`;

    if (!decorative && title == null) {
      throw new Error('[AcceptIcon]: Missing a title for non-decorative icon.');
    }

    return (
      <IconWrapper as={as} display={display} element={element} size={size} color={color} ref={ref}>
        <svg
          role="img"
          aria-hidden={decorative}
          width="100%"
          height="100%"
          viewBox="0 0 20 20"
          fill="none"
          aria-labelledby={titleId}
        >
          {title ? <title id={titleId}>{title}</title> : null}
          <path
            fill="currentColor"
            fillRule="evenodd"
            clipRule="evenodd"
            d="M16.669 6.315c.435.427.442 1.126.016 1.561L9.49 15.22a1.104 1.104 0 01-1.576 0l-3.598-3.663a1.104 1.104 0 111.575-1.546l2.81 2.86 6.407-6.539a1.104 1.104 0 011.56-.016z"
          />
        </svg>
      </IconWrapper>
    );
  }
);

AcceptIcon.displayName = 'AcceptIcon';
export {AcceptIcon};
