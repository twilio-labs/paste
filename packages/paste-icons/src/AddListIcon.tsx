/**
 * This file was automatically generated with @twilio-labs/svg-to-react
 */
import * as React from 'react';
import {useUID} from '@twilio-paste/uid-library';

import {IconWrapper} from './helpers/IconWrapper';
import type {IconWrapperProps} from './helpers/IconWrapper';

export interface AddListIconProps extends IconWrapperProps {
  title?: string;
  decorative: boolean;
}

const AddListIcon = React.forwardRef<HTMLElement, AddListIconProps>(
  ({as, display, element = 'ICON', size, color, title, decorative}, ref) => {
    const titleId = `AddListIcon-${useUID()}`;

    if (!decorative && title == null) {
      throw new Error('[AddListIcon]: Missing a title for non-decorative icon.');
    }

    return (
      <IconWrapper as={as} display={display} element={element} size={size} color={color} ref={ref}>
        <svg
          role="img"
          aria-hidden={decorative}
          xmlns="http://www.w3.org/2000/svg"
          width="100%"
          height="100%"
          fill="none"
          viewBox="0 0 20 20"
          aria-labelledby={titleId}
        >
          {title ? <title id={titleId}>{title}</title> : null}
          <path
            fill="currentColor"
            d="M2.75 2.665a.75.75 0 110 1.5.75.75 0 010-1.5zm2.75.25a.5.5 0 000 1h10a.5.5 0 000-1h-10zm8.5 8.5a.5.5 0 00-1 0v1.5h-1.5a.5.5 0 000 1H13v1.5a.5.5 0 101 0v-1.5h1.5a.5.5 0 100-1H14v-1.5z"
          />
          <path
            fill="currentColor"
            fillRule="evenodd"
            d="M13.5 8.915a4.5 4.5 0 100 9 4.5 4.5 0 000-9zm-3.5 4.5a3.5 3.5 0 117 0 3.5 3.5 0 01-7 0z"
            clipRule="evenodd"
          />
          <path
            fill="currentColor"
            d="M5 7.415a.5.5 0 01.5-.5h10a.5.5 0 010 1h-10a.5.5 0 01-.5-.5zm.5 3.5a.5.5 0 000 1h2a.5.5 0 100-1h-2zm-2.75-4.25a.75.75 0 110 1.5.75.75 0 010-1.5zM2 11.415a.75.75 0 101.5 0 .75.75 0 00-1.5 0z"
          />
        </svg>
      </IconWrapper>
    );
  }
);

AddListIcon.displayName = 'AddListIcon';
export {AddListIcon};
