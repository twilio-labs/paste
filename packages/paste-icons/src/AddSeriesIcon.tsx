/**
 * This file was automatically generated with @twilio-labs/svg-to-react
 */
import * as React from 'react';
import {useUID} from '@twilio-paste/uid-library';

import {IconWrapper} from './helpers/IconWrapper';
import type {IconWrapperProps} from './helpers/IconWrapper';

export interface AddSeriesIconProps extends IconWrapperProps {
  title?: string;
  decorative: boolean;
}

const AddSeriesIcon = React.forwardRef<HTMLElement, AddSeriesIconProps>(
  ({as, display, element = 'ICON', size, color, title, decorative}, ref) => {
    const titleId = `AddSeriesIcon-${useUID()}`;

    if (!decorative && title == null) {
      throw new Error('[AddSeriesIcon]: Missing a title for non-decorative icon.');
    }

    return (
      <IconWrapper as={as} display={display} element={element} size={size} color={color} ref={ref}>
        <svg
          role="img"
          aria-hidden={decorative}
          xmlns="http://www.w3.org/2000/svg"
          width="100%"
          height="100%"
          fill="none"
          viewBox="0 0 20 20"
          aria-labelledby={titleId}
        >
          {title ? <title id={titleId}>{title}</title> : null}
          <path
            fill="currentColor"
            fillRule="evenodd"
            d="M3 2.5a.5.5 0 00-1 0v15a.5.5 0 00.5.5h15a.5.5 0 000-1H3v-1.19l2.459-1.23a1.5 1.5 0 002.453-.575l3.645.405a1.501 1.501 0 10.03-.915l-3.644-.405a1.5 1.5 0 00-2.932.595L3 14.691V2.5zm9.646 11.146a.5.5 0 11.708.708.5.5 0 01-.708-.708zM6.5 13a.5.5 0 100 1 .5.5 0 000-1z"
            clipRule="evenodd"
          />
          <path
            fill="currentColor"
            d="M13 4.5a.5.5 0 01.5.5v1.5H15a.5.5 0 010 1h-1.5V9a.5.5 0 01-1 0V7.5H11a.5.5 0 010-1h1.5V5a.5.5 0 01.5-.5z"
          />
          <path
            fill="currentColor"
            fillRule="evenodd"
            d="M13 2.5a4.5 4.5 0 100 9 4.5 4.5 0 000-9zM9.5 7a3.5 3.5 0 117 0 3.5 3.5 0 01-7 0z"
            clipRule="evenodd"
          />
        </svg>
      </IconWrapper>
    );
  }
);

AddSeriesIcon.displayName = 'AddSeriesIcon';
export {AddSeriesIcon};
