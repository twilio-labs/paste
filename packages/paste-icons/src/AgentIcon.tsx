/**
 * This file was automatically generated with @twilio-labs/svg-to-react
 */
import * as React from 'react';
import {useUID} from '@twilio-paste/uid-library';
import {IconWrapper, IconWrapperProps} from './helpers/IconWrapper';

export interface AgentIconProps extends IconWrapperProps {
  title?: string;
  decorative: boolean;
}

const AgentIcon: React.FC<AgentIconProps> = ({as, display, size, color, title, decorative}) => {
  const titleId = `AgentIcon-${useUID()}`;

  if (!decorative && title == null) {
    throw new Error('[AgentIcon]: Missing a title for non-decorative icon.');
  }

  return (
    <IconWrapper as={as} display={display} size={size} color={color}>
      <svg role="img" aria-hidden={decorative} width="100%" height="100%" viewBox="0 0 20 20" aria-labelledby={titleId}>
        {title ? <title id={titleId}>{title}</title> : null}
        <path
          fill="currentColor"
          fillRule="evenodd"
          d="M13.484 2.483a6.845 6.845 0 013.336 5.663c.492.106.941.389 1.252.813.333.454.473 1.023.388 1.58l-.414 2.738a2.105 2.105 0 01-2.477 1.768 6.831 6.831 0 01-3.943 2.427 1.842 1.842 0 01-3.493-.835 1.85 1.85 0 011.845-1.856 1.85 1.85 0 011.821 1.555 5.758 5.758 0 002.787-1.766 2.132 2.132 0 01-.709-1.927l.414-2.738a2.104 2.104 0 011.573-1.744l-.104.03a5.785 5.785 0 00-11.564-.013A2.1 2.1 0 015.708 9.9l.415 2.743a2.133 2.133 0 01-.388 1.58 2.105 2.105 0 01-3.78-.942l-.415-2.742a2.129 2.129 0 01.387-1.58c.302-.412.734-.69 1.21-.804a6.845 6.845 0 0110.347-5.673zM9.978 15.84a.79.79 0 00-.785.796c0 .44.353.795.785.795a.79.79 0 00.785-.795.79.79 0 00-.785-.796zM3.754 9.166a1.044 1.044 0 00-.971.42 1.069 1.069 0 00-.195.794l.416 2.746a1.044 1.044 0 001.875.47c.168-.229.238-.514.195-.794l-.416-2.748a1.044 1.044 0 00-.78-.865zm13.462.42a1.044 1.044 0 00-1.876.473l-.414 2.743c-.043.28.027.565.194.793a1.044 1.044 0 001.877-.473l.414-2.743a1.069 1.069 0 00-.195-.793z"
        />
      </svg>
    </IconWrapper>
  );
};

AgentIcon.displayName = 'AgentIcon';
export {AgentIcon};
