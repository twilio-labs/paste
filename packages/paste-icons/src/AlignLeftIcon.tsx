import { useUID } from "@twilio-paste/uid-library";
/**
 * This file was automatically generated with @twilio-labs/svg-to-react
 */
import * as React from "react";

import { IconWrapper } from "./helpers/IconWrapper";
import type { IconWrapperProps } from "./helpers/IconWrapper";

export interface AlignLeftIconProps extends IconWrapperProps {
  title?: string;
  decorative: boolean;
}

const AlignLeftIcon = React.forwardRef<HTMLElement, AlignLeftIconProps>(
  ({ as, display, element = "ICON", size, color, title, decorative }, ref) => {
    const titleId = `AlignLeftIcon-${useUID()}`;

    if (!decorative && title == null) {
      throw new Error("[AlignLeftIcon]: Missing a title for non-decorative icon.");
    }

    return (
      <IconWrapper as={as} display={display} element={element} size={size} color={color} ref={ref}>
        <svg
          role="img"
          aria-hidden={decorative}
          xmlns="http://www.w3.org/2000/svg"
          width="100%"
          height="100%"
          fill="none"
          viewBox="0 0 20 20"
          aria-labelledby={titleId}
        >
          {title ? <title id={titleId}>{title}</title> : null}
          <path
            fill="currentColor"
            fillRule="evenodd"
            d="M3.5 1a.5.5 0 01.5.5v16a.5.5 0 01-1 0v-16a.5.5 0 01.5-.5zm2.333 3.4a.29.29 0 00-.277.3v2.4c0 .18.137.3.277.3h6.223a.29.29 0 00.277-.3V4.7a.29.29 0 00-.277-.3H5.833zm-1.277.3c0-.705.558-1.3 1.277-1.3h6.223c.718 0 1.277.595 1.277 1.3v2.4c0 .705-.559 1.3-1.277 1.3H5.833c-.719 0-1.277-.595-1.277-1.3V4.7zm1.277 6.1a.29.29 0 00-.277.3v2.4c0 .179.137.3.277.3h10.89a.29.29 0 00.277-.3v-2.4a.29.29 0 00-.278-.3H5.833zm-1.277.3c0-.705.558-1.3 1.277-1.3h10.89c.718 0 1.277.595 1.277 1.3v2.4c0 .705-.559 1.3-1.278 1.3H5.833c-.719 0-1.277-.595-1.277-1.3v-2.4z"
            clipRule="evenodd"
          />
        </svg>
      </IconWrapper>
    );
  },
);

AlignLeftIcon.displayName = "AlignLeftIcon";
export { AlignLeftIcon };
