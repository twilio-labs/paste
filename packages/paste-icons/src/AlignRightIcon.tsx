/**
 * This file was automatically generated with @twilio-labs/svg-to-react
 */
import * as React from 'react';
import {useUID} from '@twilio-paste/uid-library';

import {IconWrapper} from './helpers/IconWrapper';
import type {IconWrapperProps} from './helpers/IconWrapper';

export interface AlignRightIconProps extends IconWrapperProps {
  title?: string;
  decorative: boolean;
}

const AlignRightIcon = React.forwardRef<HTMLElement, AlignRightIconProps>(
  ({as, display, element = 'ICON', size, color, title, decorative}, ref) => {
    const titleId = `AlignRightIcon-${useUID()}`;

    if (!decorative && title == null) {
      throw new Error('[AlignRightIcon]: Missing a title for non-decorative icon.');
    }

    return (
      <IconWrapper as={as} display={display} element={element} size={size} color={color} ref={ref}>
        <svg
          role="img"
          aria-hidden={decorative}
          xmlns="http://www.w3.org/2000/svg"
          width="100%"
          height="100%"
          fill="none"
          viewBox="0 0 20 20"
          aria-labelledby={titleId}
        >
          {title ? <title id={titleId}>{title}</title> : null}
          <path
            fill="currentColor"
            fillRule="evenodd"
            d="M17.4 1.5a.5.5 0 01.5.5v16a.5.5 0 01-1 0V2a.5.5 0 01.5-.5zM15 7.9a.3.3 0 00.3-.3V5.2a.3.3 0 00-.3-.3H8.6a.3.3 0 00-.3.3v2.4a.3.3 0 00.3.3H15zm1.3-.3A1.3 1.3 0 0115 8.9H8.6a1.3 1.3 0 01-1.3-1.3V5.2a1.3 1.3 0 011.3-1.3H15a1.3 1.3 0 011.3 1.3v2.4zM15 14.3a.3.3 0 00.3-.3v-2.4a.3.3 0 00-.3-.3H3.8a.3.3 0 00-.3.3V14a.3.3 0 00.3.3H15zm1.3-.3a1.3 1.3 0 01-1.3 1.3H3.8A1.3 1.3 0 012.5 14v-2.4a1.3 1.3 0 011.3-1.3H15a1.3 1.3 0 011.3 1.3V14z"
            clipRule="evenodd"
          />
        </svg>
      </IconWrapper>
    );
  }
);

AlignRightIcon.displayName = 'AlignRightIcon';
export {AlignRightIcon};
