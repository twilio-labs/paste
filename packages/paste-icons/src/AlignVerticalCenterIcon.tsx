/**
 * This file was automatically generated with @twilio-labs/svg-to-react
 */
import * as React from 'react';
import {useUID} from '@twilio-paste/uid-library';

import {IconWrapper} from './helpers/IconWrapper';
import type {IconWrapperProps} from './helpers/IconWrapper';

export interface AlignVerticalCenterIconProps extends IconWrapperProps {
  title?: string;
  decorative: boolean;
}

const AlignVerticalCenterIcon = React.forwardRef<HTMLElement, AlignVerticalCenterIconProps>(
  ({as, display, element = 'ICON', size, color, title, decorative}, ref) => {
    const titleId = `AlignVerticalCenterIcon-${useUID()}`;

    if (!decorative && title == null) {
      throw new Error('[AlignVerticalCenterIcon]: Missing a title for non-decorative icon.');
    }

    return (
      <IconWrapper as={as} display={display} element={element} size={size} color={color} ref={ref}>
        <svg
          role="img"
          aria-hidden={decorative}
          xmlns="http://www.w3.org/2000/svg"
          width="100%"
          height="100%"
          fill="none"
          viewBox="0 0 20 20"
          aria-labelledby={titleId}
        >
          {title ? <title id={titleId}>{title}</title> : null}
          <path
            fill="currentColor"
            fillRule="evenodd"
            d="M10 14.3a.5.5 0 01.5.5V18a.5.5 0 11-1 0v-3.2a.5.5 0 01.5-.5zm0-6.4a.5.5 0 01.5.5v2.4a.5.5 0 11-1 0V8.4a.5.5 0 01.5-.5zm0-6.4a.5.5 0 01.5.5v2.4a.5.5 0 01-1 0V2a.5.5 0 01.5-.5z"
            clipRule="evenodd"
          />
          <path
            fill="currentColor"
            fillRule="evenodd"
            d="M6.5 4.9c-.25 0-.375.176-.375.3v2.4c0 .124.125.3.375.3h7c.25 0 .375-.176.375-.3V5.2c0-.124-.125-.3-.375-.3h-7zm-1.375.3c0-.76.659-1.3 1.375-1.3h7c.716 0 1.375.54 1.375 1.3v2.4c0 .76-.659 1.3-1.375 1.3h-7c-.716 0-1.375-.54-1.375-1.3V5.2zm-1.25 6.1c-.25 0-.375.175-.375.3V14c0 .124.125.3.375.3h12.25c.25 0 .375-.176.375-.3v-2.4c0-.124-.125-.3-.375-.3H3.875zm-1.375.3c0-.76.659-1.3 1.375-1.3h12.25c.716 0 1.375.54 1.375 1.3V14c0 .76-.659 1.3-1.375 1.3H3.875c-.716 0-1.375-.54-1.375-1.3v-2.4z"
            clipRule="evenodd"
          />
        </svg>
      </IconWrapper>
    );
  }
);

AlignVerticalCenterIcon.displayName = 'AlignVerticalCenterIcon';
export {AlignVerticalCenterIcon};
