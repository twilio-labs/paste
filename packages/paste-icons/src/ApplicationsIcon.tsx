/**
 * This file was automatically generated with @twilio-labs/svg-to-react
 */
import * as React from 'react';
import {useUID} from '@twilio-paste/uid-library';

import {IconWrapper} from './helpers/IconWrapper';
import type {IconWrapperProps} from './helpers/IconWrapper';

export interface ApplicationsIconProps extends IconWrapperProps {
  title?: string;
  decorative: boolean;
}

const ApplicationsIcon = React.forwardRef<HTMLElement, ApplicationsIconProps>(
  ({as, display, element = 'ICON', size, color, title, decorative}, ref) => {
    const titleId = `ApplicationsIcon-${useUID()}`;

    if (!decorative && title == null) {
      throw new Error('[ApplicationsIcon]: Missing a title for non-decorative icon.');
    }

    return (
      <IconWrapper as={as} display={display} element={element} size={size} color={color} ref={ref}>
        <svg
          role="img"
          aria-hidden={decorative}
          xmlns="http://www.w3.org/2000/svg"
          width="100%"
          height="100%"
          fill="none"
          viewBox="0 0 20 20"
          aria-labelledby={titleId}
        >
          {title ? <title id={titleId}>{title}</title> : null}
          <path
            fill="currentColor"
            fillRule="evenodd"
            d="M2.007 5.002a1.5 1.5 0 011.5-1.5h9a1.5 1.5 0 011.5 1.5V6.5h2.5a1.5 1.5 0 011.5 1.5v7a1.5 1.5 0 01-1.5 1.5h-9a1.5 1.5 0 01-1.5-1.5v-1.498h-2.5a1.5 1.5 0 01-1.5-1.5v-7zm1.5-.5a.5.5 0 00-.5.5v1.5h10v-1.5a.5.5 0 00-.5-.5h-9zm9.5 3h-10v4.5a.5.5 0 00.5.5h9a.5.5 0 00.5-.5v-4.5zm1 4.5v-1.5h3V15a.5.5 0 01-.5.5h-9a.5.5 0 01-.5-.5v-1.498h5.5a1.5 1.5 0 001.5-1.5zm2.5-4.502h-2.5v2.002h3V8a.5.5 0 00-.5-.5z"
            clipRule="evenodd"
          />
        </svg>
      </IconWrapper>
    );
  }
);

ApplicationsIcon.displayName = 'ApplicationsIcon';
export {ApplicationsIcon};
