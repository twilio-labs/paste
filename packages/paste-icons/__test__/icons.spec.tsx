import * as React from 'react';
import {matchers} from 'jest-emotion';
import {render} from '@testing-library/react';
import {CustomizationProvider} from '@twilio-paste/customization';
import {AgentIcon} from '../src/AgentIcon';

expect.extend(matchers);

describe('Icons', () => {
  describe('HTML attributes', () => {
    it('should have the default element name', () => {
      const {container} = render(<AgentIcon decorative />);
      expect(container.querySelector('[data-paste-element="ICON"]')).toBeInTheDocument();
    });
    it('should have a custom element name', () => {
      const {container} = render(<AgentIcon element="CUSTOM_ICON" decorative />);
      expect(container.querySelector('[data-paste-element="CUSTOM_ICON"]')).toBeInTheDocument();
    });
  });

  describe('Customization', () => {
    it('should apply custom styles to customizaed icons', () => {
      const {container} = render(
        <CustomizationProvider
          baseTheme="default"
          // @ts-expect-error global test variable
          theme={TestTheme}
          elements={{
            ICON: {borderRadius: 'borderRadius20', backgroundColor: 'colorBackgroundSuccess'},
            CUSTOM_ICON: {borderRadius: 'borderRadius20', backgroundColor: 'colorBackgroundDestructive'},
          }}
        >
          <>
            <AgentIcon decorative />
            <AgentIcon element="CUSTOM_ICON" decorative />
          </>
        </CustomizationProvider>
      );
      expect(container.querySelector('[data-paste-element="ICON"]')).toHaveStyleRule('border-radius', '4px');
      expect(container.querySelector('[data-paste-element="ICON"]')).toHaveStyleRule(
        'background-color',
        'rgb(20,176,83)'
      );
      expect(container.querySelector('[data-paste-element="CUSTOM_ICON"]')).toHaveStyleRule('border-radius', '4px');
      expect(container.querySelector('[data-paste-element="CUSTOM_ICON"]')).toHaveStyleRule(
        'background-color',
        'rgb(214,31,31)'
      );
    });
  });
});
