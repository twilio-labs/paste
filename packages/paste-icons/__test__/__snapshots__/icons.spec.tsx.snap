// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons util functions tools/actions/listIconsAction util gets all the icon names of non blocked icons 1`] = `
Array [
  "AcceptIcon",
  "AddListIcon",
  "AddSeriesIcon",
  "AgentIcon",
  "AlignLeftIcon",
  "AlignRightIcon",
  "AlignVerticalCenterIcon",
  "ArchiveIcon",
  "ArrowBackIcon",
  "ArrowDownIcon",
  "ArrowForwardIcon",
  "ArrowUpIcon",
  "ArtificialIntelligenceIcon",
  "AttachIcon",
  "AttachmentIcon",
  "AutomaticUpdatesIcon",
  "AvailableIcon",
  "BoldIcon",
  "BuildIcon",
  "BuiltInIcon",
  "BusinessIcon",
  "ButtonIcon",
  "CalendarIcon",
  "CallActiveIcon",
  "CallAddIcon",
  "CallFailedIcon",
  "CallHoldIcon",
  "CallIcon",
  "CallIncomingIcon",
  "CallOutgoingIcon",
  "CallTransferIcon",
  "CartIcon",
  "ChatIcon",
  "CheckboxCheckIcon",
  "CheckboxLineIcon",
  "CheckmarkCircleIcon",
  "ChevronDisclosureCollapsedIcon",
  "ChevronDisclosureExpandedIcon",
  "ChevronDisclosureIcon",
  "ChevronDoubleLeftIcon",
  "ChevronDoubleRightIcon",
  "ChevronDownIcon",
  "ChevronExpandIcon",
  "ChevronLeftIcon",
  "ChevronRightIcon",
  "ChevronUpIcon",
  "ClearIcon",
  "CloseCircleIcon",
  "CloseIcon",
  "CloudIcon",
  "CodeIcon",
  "CollapseIcon",
  "ColorPickerIcon",
  "ColumnIcon",
  "CommunityIcon",
  "ConnectivityAvailableIcon",
  "ConnectivityBusyIcon",
  "ConnectivityNeutralIcon",
  "ConnectivityOfflineIcon",
  "ConnectivityUnavailableIcon",
  "CopyIcon",
  "CreditCardIcon",
  "CustomIcon",
  "CustomerCareIcon",
  "DarkModeIcon",
  "DataArrayIcon",
  "DataBarChartIcon",
  "DataBooleanIcon",
  "DataLineChartIcon",
  "DataNumberIcon",
  "DataObjectIcon",
  "DataPieChartIcon",
  "DataStringIcon",
  "DataTableIcon",
  "DatabaseIcon",
  "DeleteIcon",
  "DeliveredIcon",
  "DialpadIcon",
  "DirectoryIcon",
  "DisableIcon",
  "DividerIcon",
  "DoNotIcon",
  "DocumentationIcon",
  "DownloadIcon",
  "DragHorizontalIcon",
  "DragIcon",
  "DragVerticalIcon",
  "EditIcon",
  "ElasticSIPTrunkingCapableIcon",
  "EmailIcon",
  "EmojiIcon",
  "ErrorIcon",
  "ExpandIcon",
  "ExportIcon",
  "FaxCapableIcon",
  "FeedIcon",
  "FileAudioIcon",
  "FileIcon",
  "FileImageIcon",
  "FileVideoIcon",
  "FileZipIcon",
  "FilterIcon",
  "FlagIcon",
  "FolderIcon",
  "GitIcon",
  "HeatmapIcon",
  "HideIcon",
  "HistoryIcon",
  "ImageTextIcon",
  "InboxIcon",
  "IndentDecreaseIcon",
  "IndentIncreaseIcon",
  "InformationIcon",
  "ItalicIcon",
  "LightModeIcon",
  "LinkExternalIcon",
  "LinkIcon",
  "LoadingIcon",
  "LockIcon",
  "LogInIcon",
  "LogOutIcon",
  "LogoPasteIcon",
  "LogoTwilioIcon",
  "LowerHandIcon",
  "MMSCapableIcon",
  "MenuIcon",
  "MicrophoneOffIcon",
  "MicrophoneOnIcon",
  "MinusIcon",
  "MobileIcon",
  "MoreIcon",
  "NeutralIcon",
  "NewIcon",
  "NotesIcon",
  "NotificationIcon",
  "OrderedListIcon",
  "OutOfDateIcon",
  "PauseIcon",
  "PaymentIcon",
  "PinIcon",
  "PlayIcon",
  "PlusIcon",
  "ProcessDisabledIcon",
  "ProcessDraftIcon",
  "ProcessErrorIcon",
  "ProcessInProgressIcon",
  "ProcessNeutralIcon",
  "ProcessSuccessIcon",
  "ProcessWarningIcon",
  "ProductAIAssistantsIcon",
  "ProductAPIExplorerIcon",
  "ProductAccountDashboardIcon",
  "ProductAddOnsIcon",
  "ProductAdminAccessControlIcon",
  "ProductAdminAccountsIcon",
  "ProductAdminDomainsIcon",
  "ProductAdminResoldCustomersIcon",
  "ProductAdminSSOIcon",
  "ProductAdminUsersIcon",
  "ProductAlarmsIcon",
  "ProductAssetsIcon",
  "ProductAudiencesIcon",
  "ProductAuthyIcon",
  "ProductAutopilotIcon",
  "ProductBillingIcon",
  "ProductCLIIcon",
  "ProductChannelsIcon",
  "ProductChatIcon",
  "ProductCodeExchangeCommunityIcon",
  "ProductCodeExchangePartnerIcon",
  "ProductCommsIcon",
  "ProductConnectedDevicesIcon",
  "ProductConnectionsIcon",
  "ProductContactCenterAdminIcon",
  "ProductContactCenterAssessmentsIcon",
  "ProductContactCenterQueuesIcon",
  "ProductContactCenterTasksIcon",
  "ProductContactCenterTeamsIcon",
  "ProductConversationalInsightsIcon",
  "ProductConversationsIcon",
  "ProductDebuggerIcon",
  "ProductDestinationsIcon",
  "ProductElasticSIPTrunkingIcon",
  "ProductEmailAPIIcon",
  "ProductEngageIcon",
  "ProductEngagementIntelligencePlatformIcon",
  "ProductEventLibraryIcon",
  "ProductEventStreamIcon",
  "ProductEventStreamsIcon",
  "ProductFaxIcon",
  "ProductFlexIcon",
  "ProductFlowIcon",
  "ProductFrontlineIcon",
  "ProductFunctionsIcon",
  "ProductHomeIcon",
  "ProductInsightsIcon",
  "ProductInterconnectIcon",
  "ProductInternetOfThingsEmbeddedSIMIcon",
  "ProductInternetOfThingsIcon",
  "ProductInternetOfThingsNarrowbandIcon",
  "ProductInternetOfThingsProgrammableAssetTrackerIcon",
  "ProductInternetOfThingsSuperSIMIcon",
  "ProductInternetOfThingsTrustOnboardIcon",
  "ProductInternetOfThingsWirelessIcon",
  "ProductJourneysIcon",
  "ProductKeysIcon",
  "ProductLiveIcon",
  "ProductLogsIcon",
  "ProductLookupIcon",
  "ProductMappingIcon",
  "ProductMarketingCampaignsIcon",
  "ProductMessagingIcon",
  "ProductMicrovisorIcon",
  "ProductNotifyIcon",
  "ProductOneAdminIcon",
  "ProductPayConnectorIcon",
  "ProductPersonasIcon",
  "ProductPhoneNumbersIcon",
  "ProductPrivacyIcon",
  "ProductProtocolsIcon",
  "ProductProxyIcon",
  "ProductRegionalIcon",
  "ProductReverseETLIcon",
  "ProductSDKIcon",
  "ProductSegmentIcon",
  "ProductSendGridIcon",
  "ProductSettingsIcon",
  "ProductSourceSchemaIcon",
  "ProductSourcesIcon",
  "ProductStudioIcon",
  "ProductSupportIcon",
  "ProductSwitcherIcon",
  "ProductSyncIcon",
  "ProductTaskRouterIcon",
  "ProductTraitsIcon",
  "ProductTrustHubIcon",
  "ProductTwiMLBinsIcon",
  "ProductTwilioOrgIcon",
  "ProductUSSDIcon",
  "ProductUnifyIcon",
  "ProductUsageIcon",
  "ProductVerifyIcon",
  "ProductVideoIcon",
  "ProductVoiceIcon",
  "ProductVoiceIntelligenceIcon",
  "RaiseHandIcon",
  "RecordIcon",
  "RedoIcon",
  "RefreshIcon",
  "RepeatIcon",
  "RepeatPurchaseIcon",
  "ResetIcon",
  "SMSCapableIcon",
  "SMSIcon",
  "ScreenShareIcon",
  "SearchIcon",
  "SelectIcon",
  "SelectedIcon",
  "SendIcon",
  "SentIcon",
  "SentimentNegativeIcon",
  "SentimentNeutralIcon",
  "SentimentPositiveIcon",
  "ShareIcon",
  "ShowIcon",
  "ShrinkIcon",
  "SkipBackIcon",
  "SkipForwardIcon",
  "SocialIcon",
  "SortAlphabeticalIcon",
  "SpacerVerticalIcon",
  "StarIcon",
  "StopIcon",
  "StopScreenShareIcon",
  "StoreIcon",
  "StrikethroughIcon",
  "SubscriptIcon",
  "SuccessIcon",
  "SuperscriptIcon",
  "SupportIcon",
  "SystemStatusIcon",
  "TaskIcon",
  "TextAlignCenterIcon",
  "TextAlignJustifyIcon",
  "TextAlignLeftIcon",
  "TextAlignRightIcon",
  "TextFormatClearIcon",
  "TextFormatIcon",
  "TextHighlightIcon",
  "ThemeIcon",
  "ThumbsDownIcon",
  "ThumbsUpIcon",
  "TimeIcon",
  "TipIcon",
  "TokenIcon",
  "TourIcon",
  "TransferIcon",
  "TranslationIcon",
  "TrendDownIcon",
  "TrendUpIcon",
  "UnarchiveIcon",
  "UnderlineIcon",
  "UndoIcon",
  "UnlockIcon",
  "UnorderedListIcon",
  "UnpinIcon",
  "UnsortedIcon",
  "UnsubscribeIcon",
  "UploadIcon",
  "UploadToCloudIcon",
  "UpsellIcon",
  "UserIcon",
  "UsersIcon",
  "VideoOffIcon",
  "VideoOnIcon",
  "VoiceCapableIcon",
  "VoicemailIcon",
  "VolumeOffIcon",
  "VolumeOnIcon",
  "WarningIcon",
  "WebCapableIcon",
  "WinbackIcon",
  "ZoomInIcon",
]
`;
